ludwig/models/modules/sequence_decoders.py

```diff
@@ -131,26 +145,34 @@ def __call__(
                eval_logits, train_logits, class_weights, class_biases
 
 
-class Tagger:
+class SequenceTaggerDecoder(Layer):
     def __init__(
             self,
             initializer=None,
             regularize=True,
             attention=False,
+            num_classes=0,
+            loss=None,
             **kwargs
     ):
+        super(SequenceTaggerDecoder, self).__init__()
         self.initializer = initializer
         self.regularize = regularize
         self.attention = attention
+        self.class_weights = loss['class_weights']
```

Comment:
You're right....I think this ended up here as part of my first attempt at creating the decoder.  I was playing safe and putting most stuff in the function definition incase it would be needed.  item is removed.